internal/core/tests.rs

```diff
@@ -50,6 +50,7 @@ pub extern "C" fn slint_send_keyboard_char(
 ) {
     for ch in string.chars() {
         window_adapter.window().dispatch_event(if pressed {
+            // TODO: Expose and test? Change all call sites or add a new send_repeat_keyboard_char?
```

Comment:
no need to expose there.  This is just a helper functions.
The test can send the event directly.

Reply:
- Removed the todo and added a test
- Added dispatch_key_press_repeat_event in the C++ API (just a separate function I guess, since the WindowEvent Rust API is a separate enum)

> Don't we need the same for the release event?

I wasn't sure at first but I didn't see Qt or winit send me repeated releases, the Android API states ["For key up events, this is always equal to zero."](https://developer.android.com/reference/android/view/KeyEvent#getRepeatCount()) and I think it would also be wrong to send press-release-press-release automatically if the user never released the key physically.
I'm still not 100% sure that this applies everywhere but it seems like it can be added later if needed, without requiring to introduce a breaking change. The main drawback of doing it later on top of the current state of this PR would be that maybe a separate `dispatch_key_release_repeat_event` in C++ would start looking less concise than if we added an enum/flag parameter instead.